example/data/acceptable_tekton_tasks_git.yml

```diff
@@ -0,0 +1,64 @@
+# commits for Task definitions consumed via Tekton's git resolver
+# (https://tekton.dev/docs/pipelines/git-resolver/)
+
+# Closely align with the current format of acceptable bundles.
+acceptable-task-git-refs-option1:
+  https://github.com/redhat-appstudio/build-definitions.git:
+    task/buildah/0.1/buildah.yaml:
+      - commit: 3672a457e3e89c0591369f609eba727b8e84108f
+        effective_on: 2024-02-16T00:00:00Z
+      - commit: 81185d669f543f5cd9114377f7ee0480e15a7ac1
+        effective_on: 2024-02-10T00:00:00Z
+    task/clair-scan/0.1/clair-scan.yaml:
+      - commit: 28b86c8b94c42995d98f843b0fc854c1cf2ae500
+        effective_on: 2024-02-15T00:00:00Z
+      - commit: 4efdfc8c8da9f78602b5a21bcd2b3cf5667503fc
+        effective_on: 2024-01-31T00:00:00Z
+
+# Align with the future of acceptable bundles format:
+# https://github.com/enterprise-contract/ec-cli/issues/1291
+acceptable-task-git-refs-option2:
+  https://github.com/redhat-appstudio/build-definitions.git//task/buildah/0.1/buildah.yaml:
+    - commit: 3672a457e3e89c0591369f609eba727b8e84108f
+      effective_on: 2024-02-16T00:00:00Z
+    - commit: 81185d669f543f5cd9114377f7ee0480e15a7ac1
+      effective_on: 2024-02-10T00:00:00Z
+  https://github.com/redhat-appstudio/build-definitions.git//task/clair-scan/0.1/clair-scan.yaml:
+    - commit: 28b86c8b94c42995d98f843b0fc854c1cf2ae500
+      effective_on: 2024-02-15T00:00:00Z
+    - commit: 4efdfc8c8da9f78602b5a21bcd2b3cf5667503fc
+      effective_on: 2024-01-31T00:00:00Z
+
+# Align with how the git resolver is used.
+acceptable-task-git-refs-option3:
+  - - name: url
+      value: https://github.com/redhat-appstudio/build-definitions.git
+    - name: pathInRepo
+      values: task/buildah/0.1/buildah.yaml
+    - name: revision
+      values:
+        - commit: 3672a457e3e89c0591369f609eba727b8e84108f
+          effective_on: 2024-02-16T00:00:00Z
+        - commit: 81185d669f543f5cd9114377f7ee0480e15a7ac1
+          effective_on: 2024-02-10T00:00:00Z
+  - - name: url
+      value: https://github.com/redhat-appstudio/build-definitions.git
+    - name: pathInRepo
+      values: task/clair-scan/0.1/clair-scan.yaml
+    - name: revision
+      values:
+        - commit: 28b86c8b94c42995d98f843b0fc854c1cf2ae500
+          effective_on: 2024-02-15T00:00:00Z
+        - commit: 4efdfc8c8da9f78602b5a21bcd2b3cf5667503fc
+          effective_on: 2024-01-31T00:00:00Z
+
+# The Earth is flat
+acceptable-task-git-refs-option4:
+  - ref: https://github.com/redhat-appstudio/build-definitions.git//task/buildah/0.1/buildah.yaml?ref=<sha>
+    effective_on: 2024-02-15T00:00:00Z
+  - ref: https://github.com/redhat-appstudio/build-definitions.git//task/buildah/0.1/git-clone.yaml?ref=<othersha>
+    effective_on: 2024-03-15T00:00:00Z
+  - ref: https://github.com/redhat-appstudio/build-definitions.git//task/buildah/0.1/buildah.yaml?ref=<differentsha>
+    effective_on: 2024-03-15T00:00:00Z
```

Comment:
Hmm, flat one does have some appeal IMO. Easy to read and understand. We can sort it at build time.

Reply:
Could also convert it to a nested structure at run time if we decide the scanning efficiency really does matter.

Reply:
I think a flat structure like this is going to make it really hard to make any sense of the data. Imagine this list has 100 records.

If any processing of the data requires converting the data, maybe that's an indicator the data is not in the right format?

Reply:
On the other hand, a list like this could potentially make it easier to consolidate the representation of git resolver and bundle refs.